internal/app/secretless/proxy.go

```diff
@@ -1,12 +1,12 @@
 package secretless
 
 import (
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1